tests/integration_tests/assets/echo_server.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=echo_server
+Before=cloud-init-local.service
```

Comment:
Good idea, but technically the dependencies shouldn't even matter because I'm not even doing a reboot in the test and this file isn't pushed to the instance until after it is up. The only reason this file exists is because it's the only way to run a process in the background that doesn't involve more substantial changes to pycloudlib.